components/base/BaseCollectionForm.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <h2 class="title is-size-3">
+    <h2 class="title text-3-5xl">
```

Comment:
typo?

Reply:
is-size-3 is 2rem and tailwind has text-3xl(font-size: 1.875rem) and text-4xl(font-size: 2.25rem). I opted to keep the original size of 2rem. Should I change it to 1.875rem?

Reply:
- https://tailwindcss.com/docs/font-size#customizing-your-theme
or 
- https://tailwindcss.com/docs/font-size#arbitrary-values

choose one of the solutions